fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/impl/resource/loader/FabricModResourcePack.java

```diff
@@ -47,8 +47,8 @@ public InputSupplier<InputStream> openRoot(String... pathSegments) {
 		String fileName = String.join("/", pathSegments);
 
 		if ("pack.mcmeta".equals(fileName)) {
-			String description = "Mod resources.";
-			String pack = String.format("{\"pack\":{\"pack_format\":" + SharedConstants.getGameVersion().getResourceVersion(type) + ",\"description\":\"%s\"}}", description);
+			String description = "pack.description.modResources";
+			String pack = String.format("{\"pack\":{\"pack_format\":" + type.getPackVersion(SharedConstants.getGameVersion()) + ",\"description\":{\"translate\":\"%s\",\"fallback\":\"Mod resources.\"}}}", description);
```

Comment:
```suggestion
			String fallback = "Mod resources.";
			String pack = String.format("{\"pack\":{\"pack_format\":" + type.getPackVersion(SharedConstants.getGameVersion()) + ",\"description\":{\"translate\":\"%s\",\"fallback\":\"%s\"}}}", description, fallback);
```

This is also one way to do it, if you think that's better.